appstudio-utils/util-scripts/lib/fetch/cluster.sh

```diff
@@ -13,8 +13,17 @@ k8s-save-data() {
   oc get $namespace_opt $kind $name -o json > $file
 }
 
+_policy-config-from-configmap() {
```

Comment:
using `_` to indicate a private function, wouldn't want code outside of this file to depend on this